build.xml

```diff
@@ -201,8 +201,8 @@
     <javac
         destdir="${classes}"
         includeantruntime="false"
-        target="1.8"
-        source="1.8"
+        target="11"
+        source="11"
```

Comment:
Is this the only place where target and source versions are specified?

Reply:
I had a couple of questions about `build.xml` involving that:

1. is there some reason why this is set to `1.6`?
```xml
  <target name="compile_client" description="compile only the client">
    <mkdir dir="${client_classes}"/>
    <javac destdir="${client_classes}" includeantruntime="false" target="1.6" source="1.6"
      debug="true" classpathref="compile.classpath">
```

2. is there a reason why these extra entries are included in the java docs?
```xml
    <javadoc access="protected" author="false" classpathref="compile.classpath"
      destdir="${doc}" nodeprecated="false" nodeprecatedlist="false"
      noindex="false" nonavbar="false" notree="false"
      source="1.8" splitindex="true" use="true" version="true">
      <link href="http://download.oracle.com/javase/8/docs/api/"/>
      <link href="http://fasterxml.github.io/jackson-core/javadoc/2.2.0/"/>
      <link href="http://fasterxml.github.io/jackson-databind/javadoc/2.2.0/"/>
      <fileset refid="client-jdoc-files"/>
      <fileset refid="sources-jdoc-files"/>
    </javadoc>
```
The Java SE link seems reasonable but the jackson-core and jackson-databind links are broken, which makes me wonder if they are being used.

Reply:
1. We can probably delete the compile client target altogether. I put that in there years ago and I don't really think it's needed. It was originally there because ppl were still using java 6 when we upgraded to 7

2. The are some parts of the public api that expose jackson classes and so they're in the docs to link out. Looks like the new docs are at https://www.javadoc.io/doc/com.fasterxml.jackson.core/jackson-databind/2.2.0/index.html

Reply:
Is there anything else in the build.xml file that is no longer needed?

Reply:
Probably, but the whole thing will be deleted once I get gradle working, so probably not worth spending effort on

Reply:
:+1:

Reply:
For the client, there's a makefile target that calls the ant target that'll need to be deleted, and I think the `client_classes` property in build.xml can be deleted

Reply:
Uh oh, looks like the changes to the build.xml file got nuked

Reply:
they should be back in their full glory now.

Reply:
:+1: